tools/perf/ResultsComparer/src/ResultsComparer/CommandLineOptions.cs

```diff
@@ -13,17 +13,23 @@ namespace ResultsComparer
 {
     public class CommandLineOptions

```

Comment:
@g2mula once told me that this was an old package that is no longer supported. He recommends this package here: https://github.com/dotnet/command-line-api

Reply:
I see it is not this one